server/api/jurisdictions.py

```diff
@@ -519,17 +517,19 @@ def batch_round_status(election: Election, round: Round) -> Dict[str, JSONDict]:
             Jurisdiction.name, Jurisdiction.id
         )
     )
-    assert len(list(election.contests)) == 1
-    contest = list(election.contests)[0]
-    for batch_key, audited_result in audited_results.items():
-        if batch_key in batch_keys_in_round:
-            vote_deltas = batch_vote_deltas(
-                reported_results[batch_key][contest.id], audited_result[contest.id]
-            )
-            if vote_deltas:
-                jurisdiction_name, _ = batch_key
-                jurisdiction_id = jurisdiction_name_to_id[jurisdiction_name]
-                discrepancy_count_by_jurisdiction[jurisdiction_id] += 1
+    contests = list(election.contests)
+    for contest in contests:
+        reported_results = batch_tallies(contest)
+        audited_results = sampled_batch_results(contest, include_non_rla_batches=True)
+        for batch_key, audited_result in audited_results.items():
+            if batch_key in batch_keys_in_round:
+                vote_deltas = batch_vote_deltas(
+                    reported_results[batch_key][contest.id], audited_result[contest.id]
+                )
+                if vote_deltas:
+                    jurisdiction_name, _ = batch_key
+                    jurisdiction_id = jurisdiction_name_to_id[jurisdiction_name]
+                    discrepancy_count_by_jurisdiction[jurisdiction_id] += 1
```

Comment:
The way this loop is written, if a batch has a discrepancy in both contests 1 and 2, the discrepancy count will be incremented by 2, not 1. In other words, the discrepancy count represents the number of batch-contest pairs with discrepancies, not the number of batches with discrepancies.

Reply:
That makes sense to me! Maybe leave a note in the code?

Reply:
https://github.com/votingworks/arlo/pull/1884/commits/d1fe7816759518bffcb9b50df5b134ce76da2120

Reply:
✅ https://github.com/votingworks/arlo/commit/d1fe7816759518bffcb9b50df5b134ce76da2120